Jmeter/Create Topics.jmx

```diff
@@ -0,0 +1,1009 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
may be we rename the folder Jmeter to all lowercase jmeter?

Reply:
where do we publish the bench marking results

Reply:
So the benchmarking results aren't included in this initial commit.
However you can see them in jmeter by looking at the summary report which shows 
The number of requests made to Klaw
The Error Rate
The Throughput (number of requests per second)
The size of the requests.

Total number of requests sent.